### PR TITLE
Some initial chart stuff

### PR DESCRIPTION
**Pull request by blakeparkinson**

### CODE REVIEW

**Comment by blakeparkinson** [comment]
Problem: The percantages don't really make sense for other sports (football has like 60 people, everyone will be below 20%.)

Solution: Rather than doing percentadges. We break the correct answers into buckets (ie 1-3 correct, 4-6 correct, 7-9 correct, etc...) I think all sports should be buckets divisable by 3 except for football which will be divisable by 5. The highest bucket will be 15+ for the divisables by 3. and 25+ for football.

I also think we should tell people what percentile they finished in. I dont know what the math is to calculate that.

File: public/javascripts/quiz.js [hunk 1]
```diff
@@ -84,16 +85,153 @@ var uploadScore = function(correct_answers){
...
+                label: '40%-60%'
+            },
+            {
+                value: response.all_scores.mlow,
+                color: "#46BFBD",
+                label: '20%-40%'
+
+            },
+            {
+                value: response.all_scores.mlow,
```
Comment (kjgross) [inline public/javascripts/quiz.js]: all_scores.low

**Comment by blakeparkinson** [comment]
@kjgross @hlyford11 my picasso is rdy. Can you review.merge

File: public/javascripts/quiz.js [hunk 2]
```diff
@@ -85,16 +86,215 @@ var uploadScore = function(correct_answers){
...
+        label = '9-11';
+        break;
+      case 'med':
+        label = '6-8';
+        break;
+      case 'mlow':
+        label = '3-5';
+        break;
+      case 'low':
+        label = '2 or Less';
```
Comment (kjgross) [inline public/javascripts/quiz.js]: 2 of Fewer

Comment (blakeparkinson) [inline public/javascripts/quiz.js]: why they mean the same and less takes up less space

File: public/javascripts/quiz.js [hunk 3]
```diff
@@ -85,16 +86,215 @@ var uploadScore = function(correct_answers){
...
+                label: getLegendLabel('mhigh')
+            },
+            {
+                value: response.all_scores.med,
+                color: "#B7CA85",
+                label: getLegendLabel('med')
+            },
+            {
+                value: response.all_scores.mlow,
+                color: "#46BFBD",
```
Comment (blakeparkinson) [inline public/javascripts/quiz.js]: constants for colors

File: public/javascripts/quiz.js [hunk 4]
```diff
@@ -85,16 +86,215 @@ var uploadScore = function(correct_answers){
...
+      color = '#8C33B7';
+      break;
+    case (score >= 0.1 && score < 2 ):
+      color= '#46BFBD';
+      break;
+    case (score >= 2 && score < 3):
+      color= '#B7CA85';
+      break;
+    case (score >= 3 && score < 4 ):
+      color= '#FFFF59';
```
Comment (blakeparkinson) [inline public/javascripts/quiz.js]: constants

**Comment by blakeparkinson** [comment]
simple/beautiful. tears of joy

**Pull request merged**